src/ripple/protocol/Feature.h

```diff
@@ -369,6 +369,7 @@ extern uint256 const fixXahauV3;
 extern uint256 const fix20250131;
 extern uint256 const featureHookCanEmit;
 extern uint256 const fixRewardClaimFlags;
+extern uint256 const featureClawback;
```

Comment:
Logic error: More features defined than allocated. Adjust numFeatures in Feature.h.